### PR TITLE
Adding streaming_service property

### PR DESCRIPTION
**Pull request by ratoaq2**
Adding `streaming_service` property as discussed in #344

### CODE REVIEW

File: guessit/rules/properties/streaming_service.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    :rtype: Rebulk
+    """
+    rebulk = Rebulk().string_defaults(ignore_case=True)
+    rebulk.defaults(name='streaming_service', validator=seps_surround)
+
+    rebulk.string('AE', value='A&E')
+    rebulk.string('AMBC', value='ABC')
+    rebulk.string('AMZN', value='Amazon Prime')
+    rebulk.string('AS', value='Adult Swim')
+    rebulk.string('iP', value='BBC iPlayer')
```
Comment (OmgImAlexis) [inline guessit/rules/properties/streaming_service.py]: There's also ABC iPlayer which is for all the Australian ABC channels and is where 99% of Australian kids shows are ripped from.

Comment (duramato) [inline guessit/rules/properties/streaming_service.py]: iP is BBC iPlayer

Comment (OmgImAlexis) [inline guessit/rules/properties/streaming_service.py]: Yeah..? That is the BBC(UK) one, not the Australian one.

Comment (duramato) [inline guessit/rules/properties/streaming_service.py]: Yes all releases with the iP are from uk's bbc streaming service.

Comment (OmgImAlexis) [inline guessit/rules/properties/streaming_service.py]: I know I was saying that the other one needs to be added, I'm not sure about the tag for it though.

Comment (Toilal) [inline guessit/rules/properties/streaming_service.py]: Can you give an example with ABC iPlayer @OmgImAlexis  ? What's the tag in this case ?

Comment (OmgImAlexis) [inline guessit/rules/properties/streaming_service.py]: iView seems to be the main tag used. For example `The.Show.S01.E01.iView.Webrip.BLINKMAN.mp4`

Comment (ratoaq2) [inline guessit/rules/properties/streaming_service.py]: This one is interesting. All the others have a convention:
- Up to 4 letters (Usually all uppercase) 
- before `format`
- after `screen_size` whenever possible

This one is 5 letters and not uppercase. Anyway, that's just a convention, not a rule

Comment (OmgImAlexis) [inline guessit/rules/properties/streaming_service.py]: @ratoaq2 don't expect anything todo with media and Australia to follow standards, haha

**Comment by Toilal** [comment]
LGTM ! But don't forget docs next time :)

**Pull request merged**